torch/nn/functional.py

```diff
@@ -465,14 +466,19 @@ def lp_pool2d(input, norm_type, kernel_size, stride=None, ceil_mode=False):
 
 
 def lp_pool1d(input, norm_type, kernel_size, stride=None, ceil_mode=False):
+    # type: (Tensor, int, int, Optional[List[int]], bool) -> Tensor
     r"""Applies a 1D power-average pooling over an input signal composed of
     several input planes. If the sum of all inputs to the power of `p` is
     zero, the gradient is set to zero as well.
 
     See :class:`~torch.nn.LPPool1d` for details.
     """
-    out = avg_pool1d(input.pow(norm_type), kernel_size, stride, 0, ceil_mode)
-    return (torch.sign(out) * relu(torch.abs(out))).mul(kernel_size).pow(1. / norm_type)
+    if stride is None:
+        _stride = torch.jit.annotate(List[int], [])
+    else:
+        _stride = torch.jit._unwrap_optional(stride)
+    out = avg_pool1d(input.pow(norm_type), kernel_size, _stride, 0, ceil_mode)
+    return (torch.sign(out) * relu(torch.abs(out))).mul(kernel_size).pow(1 / norm_type)
```

Comment:
Was there a reason it was changed from 1. to 1 ? I don't know if that will have different behavior, esp with regards to python 2/3. I have a diff to binary ops between ints & floats landing now.